swarm_init_pos:
        [1.0898,  0.1225, 0.0502,
         -0.0887,  1.1686, 0.0615,
         -0.8997, 0.1321, 0.0247,
         0.0893, -0.9071, 0.0267]